core/divelogexportlogic.cpp

```diff
@@ -128,21 +128,19 @@ void exportHtmlInitLogic(const QString &filename, struct htmlExportSetting &hes)
 	QFile file(filename);
 	QFileInfo info(file);
 	QDir mainDir = info.absoluteDir();
-	mainDir.mkdir(file.fileName() + "_files");
-	QString exportFiles = file.fileName() + "_files";
+	const QString exportFiles = file.fileName() + "_files" + QDir::separator();
+	mainDir.mkdir(exportFiles);
 
-	QString json_dive_data = exportFiles + QDir::separator() + "file.js";
-	QString json_settings = exportFiles + QDir::separator() + "settings.js";
-	QString translation = exportFiles + QDir::separator() + "translation.js";
-	QString stat_file = exportFiles + QDir::separator() + "stat.js";
-	exportFiles += "/";
+	const QString json_dive_data = exportFiles + "file.js";
```

Comment:
Is there any point in making these const? We generally don't follow a const-by-default style for local variables.

Reply:
I'm just used to that.